### ISSUE TITLE
Add screenshots to README.md

### ISSUE THREAD

**Issue by gayashanbc** [scorelab/Bassa-mobile#39]
### Description

The goal of this task is to add screenshots of the various functionalities of the app to the README.md file

**Comment by Susith1995** [comment]
is it still open,can I take up this issue?

**Comment by JaDogg** [comment]
@Susith1995 Yes run it and add more screenshots to the README. It would be better if it is a GIF

**Comment by gayashanbc** [comment]
@Susith1995 Yes, feel free to.
+1 for a GIF.

**Comment by karthikshetty03** [comment]
If this issue is still not resolved, can I work on this ?

**Comment by gayashanbc** [comment]
> If this issue is still not resolved, can I work on this ?

Sure, go ahead
As mentioned in the above comments, a GIF of several screenshots would be ideal.
🙂

**Comment by karthikshetty03** [comment]
![Screenshot from 2020-08-05 16-59-24](https://user-images.githubusercontent.com/45932358/89407979-522ed800-d73d-11ea-89e4-ee3bac925ce8.png)

I am getting this error while setting up the project on my local machine
Also, it gets fixed and the backend server works fine if I change the import statement everywhere from  

**from flask.ext.cors import CORS** to 
**from flask_cors import CORS**

![Screenshot from 2020-08-05 17-04-44](https://user-images.githubusercontent.com/45932358/89408832-aa1a0e80-d73e-11ea-8807-06bab6f60e8a.png)

How should I proceed with this ?

**Comment by gayashanbc** [comment]
how are you doing today?
my name is Gayashan Kalhara Bombuwala

**Comment by veerviren** [comment]
@gayashanbc I like to work on this issue, if it is still open,s please assign it to me

**Comment by gayashanbc** [comment]
> @gayashanbc I like to work on this issue, if it is still open,s please assign it to me

Assigned it to you

**Comment by AtrikeshM** [comment]
I want to work on this issue. Please assign me.

**Comment by Smriti925** [comment]
If it is still open please assign it to me.
